devtools/create_project/msvc10/create_project.vcxproj.filters

```diff
@@ -53,6 +56,9 @@
     <ClCompile Include="..\xcode.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
+    <ClCompile Include="..\cmake.cpp">
```

Comment:
Same as above, where is cmake being used?

Reply:
create_project has a CMakeProvider generator (since 2016 according to Git). Even if it's not used, it needs to be included for the project to compile.